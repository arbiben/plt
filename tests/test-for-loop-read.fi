$ this test file tests looping through an array of files names, for each file name, it read the file contents and prints them out$
$ Does not exist in MicroC $
$ Expects the following output: $

$
Hello I am a sentence composed out of ten files
$



int i?
str tmp?
arr str 5 files?
files[0] = "file1.txt"?
files[1] = "file2.txt"?
files[2] = "file3.txt"?
files[3] = "file4.txt"?
files[4] = "file5.txt"?
files[5] = "file6.txt"?
files[6] = "file7.txt"?
files[7] = "file8.txt"?
files[8] = "file9.txt"?
files[9] = "file10.txt"?


int main(){

for(i = 0; i<10;i++)
{
    tmp = readFile(files@i)?
    printstring(tmp)?
    printstring(" ")?
}
