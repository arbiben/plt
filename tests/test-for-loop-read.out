Hello 
I 
am 
a 
sentence 
composed 
out 
of 
ten 
files
